src/test/java/com/atlassian/db/replica/it/example/aurora/replica/AuroraConnectionProvider.java

```diff
@@ -32,9 +31,10 @@ public Connection getReplicaConnection() throws SQLException {
     }
 
     private Connection getConnection(String url) throws SQLException {
-        final Properties props = new Properties();
-        props.setProperty("user", "postgres");
-        props.setProperty("password", System.getenv("password"));
-        return DriverManager.getConnection(url, props);
+        return DriverManager.getConnection(
+            url,
+            "postgres",
+            System.getenv("password")
```

Comment:
this is strange. I think product should determine the way pass password
It also complicate testing

Reply:
Ah sorry 🤦 this is test implemention. All good